### DISCUSSION TITLE
Fiemware Update

### DISCUSSION THREAD

**Discussion by richdon1** [kachurovskiy/nanoels#187] [Q&A] [answered]
Hi, This is my first post. I saw that the firmware on the h4 nanoels is up to V12.  Silly question but, how do you update the fitmware? Sorry, complete newb.
Don

**Comment by kachurovskiy** [comment]
Hi @richdon1 - just follow the steps on https://github.com/kachurovskiy/nanoels/tree/main/h4#programming-the-controller carefully. Thanks!

**Reply by richdon1** [reply, marked as answer]
So, basically upload the new sketch. Got it. Thank you,
Don
